elixir/apps/domain/lib/domain/auth/identity/sync.ex

```diff
@@ -66,18 +67,27 @@ defmodule Domain.Auth.Identity.Sync do
         identities,
         {provider_identifiers, [], []},
         fn identity, {insert, update, delete} ->
-          if identity.provider_identifier in provider_identifiers do
-            {
-              insert -- [identity.provider_identifier],
-              [identity.provider_identifier] ++ update,
-              delete
-            }
-          else
-            {
-              insert -- [identity.provider_identifier],
-              update,
-              [identity.provider_identifier] ++ delete
-            }
+          cond do
+            identity.provider_identifier in provider_identifiers ->
+              {
+                insert -- [identity.provider_identifier],
```

Comment:
Would it make sense to pull all of these `insert -- [identity.provider_identifier]` lines out above the `cond` block since it happens in all of them?

Reply:
Yeah, that is a good idea, will push in a sec